examples/README.md

```diff
@@ -123,3 +124,27 @@ Got  Expected
 Hits: 4, Misses: 0
 Accuracy: 100%
 ```
+
+### Naïve Bayes
+
+#### Dog Classification
+
+Suppose we have a population composed of red dogs and white dogs, whose friendliness, furriness, and speed can be measured. The group of white dogs is friendlier, furrier, and slower than red dogs by one standard deviation (respectively), but given the color of a dog, friendliness, furriness, and speed are independent of each other. We can use a naïve Bayes model to try to predict dog color given friendliness, furriness, and speed as observations.
```

Comment:
I think another paragraph with a little more technical detail would be good. In particular I think it's worthwhile to address:

- What kind of Naive Bayes' do we use? Why?
- As the example is quite large - maybe walk through roughly what we do? Randomly generate some dogs, convert them into `Matrix` form for rusty-machine. Create model, train model, predict from model on unknown dogs. Assess accuracy.

What do you think? Perhaps there is already enough information there for new users - it certainly makes sense to me but I'm not a good reference point!